Add cloud note to schema package (#2404)
influxdata/docs-v2: content/influxdb/cloud/reference/flux/stdlib/influxdb-schema/_index.md

weight: 202
influxdb/v2.0/tags: [functions, schema, package]
introduced: 0.88.0
cascade:
  append:
    block: cloud
    content: |
      #### Supported in the InfluxDB Cloud UI

>       The `schema` package can retrieve schema information from InfluxDB Cloud when
>       used in the InfluxDB Cloud user interface (UI), but it **cannot** when using
>       the [Flux REPL](/influxdb/cloud/tools/repl/).
>       In the REPL, there is no way to provide the host, organization, and token
>       required to authorize with InfluxDB Cloud.

Reviewer: ```suggestion
           The `schema` package can retrieve schema information from the InfluxDB Cloud user interface (UI), but **not** from the [Flux REPL](/influxdb/cloud/tools/repl/).
```
